### ISSUE TITLE
Add support for chrono Weekday

### ISSUE THREAD

**Issue by yuki0418** [juhaku/utoipa#1213]
Looks like chrono::Weekday not supported yet.
When I use Weekday, it returns error below
```
the trait bound `chrono::Weekday: utoipa::ToSchema` is not satisfied
the following other types implement trait `utoipa::ToSchema`
```

I assume that just add support like below PR?
#404

**Comment by juhaku** [comment]
> Looks like chrono::Weekday not supported yet so I may create PR to add support like below PR?

Yes, it is not supported as of now, you can create a PR like that to support it. alternatively you can alias it globally with `utoipa-config` or locally via `value_type=...` attribute. See example here https://github.com/juhaku/utoipa/issues/1205#issuecomment-2478694544
